util.go

```diff
@@ -148,6 +166,10 @@ func addFileReader(w *multipart.Writer, f *File) error {
 	return writeMultipartFormFile(w, f.ParamName, f.Name, f.Reader)
 }
 
+func addCustomDataReader(w *multipart.Writer, cd *MultipartCustomData) error {
```

Comment:
@Asker80 Please remove it, this method not needed after you take care of comments.